packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -156,29 +245,54 @@ class CupertinoTabScaffold extends StatefulWidget {
 }
 
 class _CupertinoTabScaffoldState extends State<CupertinoTabScaffold> {
-  int _currentPage;
+  CupertinoTabController _controller;
 
   @override
   void initState() {
     super.initState();
-    _currentPage = widget.tabBar.currentIndex;
+    _updateTabController();
+  }
+
+  void _updateTabController() {
+    final CupertinoTabController newController =
+      // User provided a new controller, update [_controller] with it.
```

Comment:
You don't have to wrap with [] if it's just a code comment. Nothing will link this to anything unlike dartdocs.